sdk/nodejs/README.md

```diff
@@ -1,18 +1,3 @@
-The Microsoft Azure Active Directory (Azure AD) provider package offers support for all Azure AD services and their properties.
-Services are exposed as types such as `Application`, `Group`, and `User`, among many others. Using this
-package allows you to programmatically declare instances of any Azure AD services using infrastructure as code, which
-Pulumi then uses to drive the Azure REST API.
-
-This package is typically used in conjunction with the [Pulumi Azure provider](https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/azure/).
-
-If this is your first time using this package, these two resources may be helpful:
-
-* [Azure Getting Started Guide](https://www.pulumi.com/docs/quickstart/azure): Get up and running quickly.
-* [Azure Pulumi Setup Documentation](https://www.pulumi.com/docs/quickstart/azure/configure/): How to configure Pulumi
-    for use with your Azure account.
-
-Use the navigation below to see detailed documentation, including sample code, for each of the supported Azure services.
-
```

Comment:
We shouldn't use this. Did you delete the original `README.md` when generating these changes?

Reply:
(same feedback for the other provider PRs)

Reply:
yes, I deleted the entire tree - does this need to be manually managed? If so, should it be in all parts of our SDK?

Reply:
The root-level `README.md` does need to be manually managed, yes, but the nested readmes are generated.

Reply:
I believe it's actually intentional that the nested READMEs are being deleted.  Per https://github.com/pulumi/home/issues/734#issuecomment-606323575.